bundles/mapping/mapwfs2/domain/WfsLayerModelBuilder.js

```diff
@@ -50,7 +50,7 @@ Oskari.clazz.define(
                 toolOwnStyle.setIconCls('show-own-style-tool');
                 toolOwnStyle.setTooltip(locOwnStyle);
                 toolOwnStyle.setCallback(function () {
-                    me.sandbox.postRequestByName('ShowOwnStyleRequest', [layer.getId(), undefined, false]);
+                    me.sandbox.postRequestByName('ShowUserStylesRequest', [layer.getId(), false]);
                 });
                 layer.addTool(toolOwnStyle);
```

Comment:
Now that the editor has been separated to another bundle I think we should check if the ShowUserStylesRequest request is available in the application and only then create the tool for layers.